package.json

```diff
@@ -41,30 +42,25 @@
     "backbone": "^1.3.3",
     "core-js": "^2.4.1",
     "es6-promise": "^4.0.3",
-    "kinvey-phonegap-sdk": "3.2.2",
-    "kinvey-node-sdk": "3.2.2",
-    "lodash": "^4.16.2"
-  },
-  "peerDependencies": {
-    "backbone": "^1.3.3",
-    "core-js": "^2.4.1",
-    "es6-promise": "^4.0.3",
-    "kinvey-phonegap-sdk": "3.2.2",
-    "kinvey-node-sdk": "3.2.2",
-    "lodash": "^4.16.2"
+    "kinvey-node-sdk": "3.4.0",
```

Comment:
@thomasconner are you sure this is correct? Shouldn't one of these two (node or phonegap) be sufficient? In fact, does phonegap make sense here given that most backbone apps will run in the web browser?